katana/config/newtestcase.xml

```diff
@@ -7,11 +7,13 @@
 		<Time></Time>
```

Comment:
Is the file required ?

Reply:
Hi @santhosh-warrior 
This file is already presented. I  added the tag for wrapper file.